src/cycax/cycad/Slot.py

```diff
@@ -0,0 +1,53 @@
+from Location import Location
+from Holes import Holes
+from Rectange_Cut_Out import Rectangle_Cut_Out
+
+
+class Slot:
+    hole_left: Holes
+    hole_right: Holes
+    rectangle: Rectangle_Cut_Out
+    depth: float = 2
+
+    def __init__(
+        self,
+        side: str,
+        x_size: float,
+        y_size: float,
+        z_size: float,
+        x: float,
+        y: float,
+        z: float,
+        Horizontal: bool = True,
+    ):
+        """This method will be used to define a slot with all ist parameters. To get a verticle slot enter the details in exactly the same manner to the horizontle slot and the specify that Horrizontal to False."""
+        if Horizontal:
+            self.hole_left = Holes(side=side, x=x, y=y, z=z, big=y_size, depth=z_size)
+            self.hole_right = Holes(
+                side=side, x=x + x_size, y=y, z=z, big=y_size, depth=z_size
+            )
+            self.rectangle = Rectangle_Cut_Out(
+                side=side, width=x_size, x=x, y=y, z=z, height=y_size, depth=z_size
+            )
+        else:
+            self.hole_left = Holes(side=side, x=x, y=y, z=z, big=y_size, depth=z_size)
+            self.hole_right = Holes(
+                side=side, x=x, y=y + y_size, z=z, big=y_size, depth=z_size
+            )
+            self.rectangle = Rectangle_Cut_Out(
+                side=side, width=y_size, x=x, y=y, z=z, height=x_size, depth=z_size
+            )
+
+    def export(self):
+        """This will be used to create a dict of the slot."""
+        dict_hole_left = self.hole_left.export()
+        dict_hole_right = self.hole_right.export()
+        dict_rectangle = self.rectangle.export()
+
+        return dict_hole_left, dict_hole_right, dict_rectangle
```

Comment:
Suddenly export returns a tuple and not a dict. This will get users of the library very confused.